src/main/java/ch/saunah/saunahbackend/dto/UserResponse.java

```diff
@@ -0,0 +1,87 @@
+package ch.saunah.saunahbackend.dto;
+
+import ch.saunah.saunahbackend.model.User;
+import ch.saunah.saunahbackend.model.UserRole;
+
+/**
+ * This class is used as the response DTO object, when user data was retrieved.
+ */
+public class UserResponse {
+    private int id;
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String passwordHash;
```

Comment:
I think we could leave `passwordHash` and `resetpasswordHash` away in this class?

Reply:
Done